x/tokenomics/keeper/update_relay_mining_difficulty_test.go

```diff
@@ -0,0 +1,288 @@
+package keeper_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	keepertest "github.com/pokt-network/poktroll/testutil/keeper"
+	"github.com/pokt-network/poktroll/x/tokenomics/keeper"
+	tokenomicskeeper "github.com/pokt-network/poktroll/x/tokenomics/keeper"
+	"github.com/pokt-network/poktroll/x/tokenomics/types"
+)
+
+func TestUpdateRelayMiningDifficulty_General(t *testing.T) {
+	keeper, ctx := keepertest.TokenomicsKeeper(t)
+
+	// Introduce svc1 for the first time
+	relaysPerServiceMap := map[string]uint64{
+		"svc1": 1e3, // new service
+	}
+	err := keeper.UpdateRelayMiningDifficulty(ctx, relaysPerServiceMap)
+	require.NoError(t, err)
+
+	// The first time svc1 difficulty is updated, the relay EMA will be equal
+	// to the first value provided.
+	difficultySvc11, found := keeper.GetRelayMiningDifficulty(ctx, "svc1")
```

Comment:
```suggestion
	difficultySvc1Update1, found := keeper.GetRelayMiningDifficulty(ctx, "svc1")
```
The numbers seem arbitrary at first glance.